client/src/main/java/io/pravega/client/netty/impl/ClientConnectionImpl.java

```diff
@@ -63,6 +63,7 @@ public void sendAsync(WireCommand cmd, CompletedCallback callback) {
             nettyHandler.setRecentMessage();
 
             channel = nettyHandler.getChannel();
+            log.debug("Write and flush message {} on channel {}", cmd, channel);
```

Comment:
This should be trace.

Reply:
This log at debug level was useful to verify the time at which the readers and RawClient sent out the request. This would help us understand the amount of time required by the server to respond to it. 
(The server at debug level logs all the requests it has received, similarly we could log all the requests here (or at CommandEncoder) at debug.)